.gitignore

```diff
@@ -1,3 +1,8 @@
 .vscode
 *.tar.gz
-__pycache__
\ No newline at end of file
+__pycache__
+# Ignore specific Ansible files
+playbook.yml
+inventory.ini
+ansible.cfg
```

Comment:
These can't be here as it will prevent us pushing any playbooks to the repo or any other files matching these. Please remove.